web/tests/functional/review_status/test_review_status.py

```diff
@@ -652,16 +652,55 @@ def test_review_status_changes(self):
 
         reports2 = get_all_run_results(self._cc_client, runid2)
 
+        null_deref_report2 = next(filter(
+            lambda r: r.bugHash == NULL_DEREF_BUG_HASH,
+            reports2))
         uncommented_report2 = next(filter(
             lambda r: r.bugHash == UNCOMMENTED_BUG_HASH,
             reports2))
 
+        multi_confirmed_report2 = next(filter(
+            lambda r: r.bugHash == MULTI_REPORT_HASH and
+            r.reviewData.status == ReviewStatus.CONFIRMED,
+            reports2))
+        multi_intentional_report2 = next(filter(
+            lambda r: r.bugHash == MULTI_REPORT_HASH and
+            r.reviewData.status == ReviewStatus.INTENTIONAL,
+            reports2))
+        multi_unreviewed_report2 = next(filter(
+            lambda r: r.bugHash == MULTI_REPORT_HASH and
+            r.reviewData.status == ReviewStatus.UNREVIEWED,
+            reports2), None)
+        multi_false_positive_report2 = next(filter(
+            lambda r: r.bugHash == MULTI_REPORT_HASH and
+            r.reviewData.status == ReviewStatus.FALSE_POSITIVE,
+            reports2))
+
+        self.assertIn(
+            (null_deref_report2.reviewData.status,
+             null_deref_report2.reviewData.comment),
+            map(lambda x:
+                (x['review_status'], x['review_status_comment']),
+                suppress_project_bugs[null_deref_report2.bugHash]))
```

Comment:
Ugh... could we not make a new test function for these?

Reply:
I agree with @Szelethus  on this. If we ever have a regression it will much easier to see what went wrong.
Otherwise LGTM

Reply:
Sorry, I don't understand this comment. This test is a continuation of a previous storage in the same test function. Here I test if the review statuses chance properly which were stored in a previous storage event. If I replaced these assertions to a different test function, then I couldn't ensure that it's executed after this current function.